py4j-python/src/py4j/tests/java_dir_test.py

```diff
@@ -0,0 +1,162 @@
+from __future__ import unicode_literals, absolute_import
+
+
+from py4j.java_gateway import JavaGateway, GatewayParameters, java_import,\
+    UserHelpAutoCompletion
+from py4j.tests.java_gateway_test import (
+    start_example_app_process)
+from contextlib import contextmanager
+from nose.tools import eq_
```

Comment:
let's stick to self.assertEqual or copy the eq_ code from nose: I try to keep Py4J dependency-free (even though the recommended way to run the test suite is with nose, this is not strictly required).